tests/test_tinydb_manager_for_parallel_requests.py

```diff
@@ -0,0 +1,96 @@
+import os
```

Comment:
Can you add a source code header?  Example: https://github.com/geopython/pygeoapi/blob/master/tests/test_util.py#L1-L28